zone.go

```diff
@@ -141,14 +234,56 @@ func (api *API) PurgeCache(zoneID string, pcr PurgeCacheRequest) (PurgeCacheResp
 
 // DeleteZone deletes the given zone.
 // API reference:
-// https://api.cloudflare.com/#zone-delete-a-zone
-// DELETE /zones/:id
-func DeleteZone() {
+//   https://api.cloudflare.com/#zone-delete-a-zone
+//   DELETE /zones/:id
+func (api *API) DeleteZone(zoneID string) (ZoneID, error) {
+	res, err := api.makeRequest("DELETE", "/zones"+zoneID, nil)
+	if err != nil {
+		return ZoneID{}, errors.Wrap(err, errMakeRequestError)
+	}
+	var r ZoneIDResponse
+	err = json.Unmarshal(res, &r)
+	if err != nil {
+		return ZoneID{}, errors.Wrap(err, errUnmarshalError)
+	}
+	return r.Result, nil
+}
+
+// AvailablePlans returns information about all plans available to the specified zone.
+// API reference:
+//  https://api.cloudflare.com/#zone-plan-available-plans
+//  GET /zones/:zone_identifier/available_plans
+func (api *API) AvailablePlans(zoneID string) ([]ZonePlan, error) {
```

Comment:
That does give a bit more context. I'll change that. Same for the next one. Thanks.